test/spec/lib/parser/rule.js

```diff
@@ -70,6 +70,16 @@ describe('Rule', function() {
 
     });
 
+    it('properly handles computed member expressions', function() {
```

Comment:
Is this test required? it seems redundant with the one defined in "fixtures.json"